shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/MockRequestRecordMapper.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.admin.mapper;
+
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Param;
+import org.apache.shenyu.admin.model.entity.MockRequestRecordDO;
+import org.apache.shenyu.admin.validation.ExistProvider;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This is Mock Request Record Mapper.
+ */
+@Mapper
+public interface MockRequestRecordMapper extends ExistProvider {
+
+    /**
+     * exiated.
+     *
+     * @param id id
+     * @return existed
+     */
+    @Override
+    Boolean existed(@Param("id") Serializable id);
+
+    /**
+     * Select by primary key.
+     *
+     * @param id primary key
+     * @return the value of {@linkplain MockRequestRecordDO}
+     */
+    MockRequestRecordDO queryById(String id);
+
+    /**
+     * Select by condition.
+     *
+     * @param mockRequestRecordDO condition
+     * @return The value of {@linkplain List}
+     */
+    List<MockRequestRecordDO> queryAll(@Param("item") MockRequestRecordDO mockRequestRecordDO);
+
+    /**
+     * Count with condition.
+     *
+     * @param mockRequestRecordDO condition
+     * @return The value of count result
+     */
+    long count(MockRequestRecordDO mockRequestRecordDO);
+
+    /**
+     * Insert record.
+     *
+     * @param mockRequestRecordDO record
+     * @return The value of insert count
+     */
+    int insert(MockRequestRecordDO mockRequestRecordDO);
+
```

Comment:
need to support `insertSelective`.

Reply:
![6fef1ed930e89e55da32edd3dbabb36](https://user-images.githubusercontent.com/32192263/201524994-547c340f-d9da-43fe-8fe8-1279db618314.png)

hi, @dragon-zhang, Since all the fields in the `mock_request_record` table are non-empty, I omitted this method